Makefile

```diff
@@ -17,29 +17,30 @@ MKDIR    ?= mkdir
 REGISTRY ?= nvidia
 
 DCGM_VERSION   := 2.2.9
-GOLANG_VERSION := 1.14.2
+GOLANG_VERSION := 1.17
 VERSION        := 2.4.1
 FULL_VERSION   := $(DCGM_VERSION)-$(VERSION)
 
-NON_TEST_FILES  := pkg/dcgm.go pkg/gpu_collector.go pkg/parser.go pkg/pipeline.go pkg/server.go pkg/system_info.go pkg/types.go pkg/utils.go pkg/kubernetes.go pkg/main.go
+NON_TEST_FILES  := pkg/dcgm.go pkg/gpu_collector.go pkg/parser.go pkg/pipeline.go pkg/server.go pkg/system_info.go pkg/types.go pkg/utils.go pkg/kubernetes.go cmd/dcgm-exporter/main.go
```

Comment:
pkg/dcgm.go and all the others need to be updated here to pkg/dcgmexporter/dcgm.go etc.

Reply:
Done